### PR TITLE
odo experimental mode documentation.

### PR DESCRIPTION
**Pull request by yhontyk**

### CODE REVIEW

File: cli_reference/developer_cli_odo/accessing-technology-preview-features.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
+[id='accessing-technology-preview-features']
+= Accessing Technology Preview features
+include::modules/developer-cli-odo-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: accessing-technology-preview-features
+toc::[]
+
+In `{odo-title}` you can access Technology preview in features, such as devfile support, by enabling the Experimental Mode.
```
Comment (prietyc123) [inline cli_reference/developer_cli_odo/accessing-technology-preview-features.adoc:8]: AFAIU devfile is no more under tech preview for 2.0 release. @kadel correct me if I am wrong.

Comment (kadel) [reply cli_reference/developer_cli_odo/accessing-technology-preview-features.adoc:8]: yes with odo 2.0 release devfile support in odo will no longer be a tech preview, but full-fledged supported feature

Comment (prietyc123) [reply cli_reference/developer_cli_odo/accessing-technology-preview-features.adoc:8]: Also IMO we need to check for similar occurrence in other places too. cc_ @boczkowska

Comment (boczkowska) [reply cli_reference/developer_cli_odo/accessing-technology-preview-features.adoc]: @prietyc123 removed (including other instances). 
Is the table listing the behaviour of the commands still correct?

**Pull request closed without merging**